README.md

```diff
@@ -15,6 +15,16 @@ This information reflects the head of this branch.
 | ------------------------------------------------------------------------------------------ | ---------------------------------------| --------------- |
 | [CSI Spec v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | quay.io/k8scsi/csi-provisioner:v1.0.1  | 1.13            |
 
+## Feature status
+
```

Comment:
Also should we say the provisioner itself is GA?

Reply:
Added a note below the table